gratipay/utils/emails/queue_branch_email.py

```diff
@@ -7,6 +7,23 @@
 from gratipay import wireup
 
 
+def get_recently_active_participants(db):
+    return db.all("""
+        SELECT p.*::participants
+          FROM participants p
+         WHERE email_address is not null
+           AND claimed_time  is not null
+           AND is_closed     is not true
+           AND is_suspicious is not true
+           AND ( SELECT count(*)
+                   FROM exchanges e
+                  WHERE e.participant = p.username
+                    AND timestamp > now() - '180 days'::interval
```

Comment:
It's same as 
```
NOW() - INTERVAL '180 days'
```
? Do we have preference here? Excuse me about rusty postgresql.